deployments/prod/environment.py

```diff
@@ -613,10 +784,10 @@ def mkdict(items):
 
 def env() -> Mapping[str, Optional[str]]:
     """
-    Returns a dictionary that maps environment variable names to values. The 
-    values are either None or strings. String values can contain references to 
-    other environment variables in the form `{FOO}` where FOO is the name of an 
-    environment variable. See 
+    Returns a dictionary that maps environment variable names to values. The
+    values are either None or strings. String values can contain references to
+    other environment variables in the form `{FOO}` where FOO is the name of an
+    environment variable. See
```

Comment:
Typically, a hotfix should be minimal, and not introduce cosmetic or other unrelated changes. I'll bump the review count but let this one slip through due to the time constraints.